packages/sdk-client/src/domain/domain-interface.ts

```diff
@@ -32,6 +33,13 @@ export interface UnifiedCredentials {
   conversationRegion?: ConversationRegion;
 }
 
+export interface MailgunCredentials {
+  /** Your API Key created from the [Mailgun Dashboard](https://app.mailgun.com/settings/api_security) */
+  mailgunApiKey: string;
+  /** The region for the Mailgun API. Default region is empty */
```

Comment:
Could have sense to have details about "default value is empty" and "default region is ..".

Reply:
Indeed. Fixed